frontend/src/metabase/home/homepage/types.ts

```diff
@@ -0,0 +1,29 @@
+export interface User {
+  first_name: string;
+  is_superuser: boolean;
+  personal_collection_id: string;
+}
+
+export interface Database {
+  id: number;
+  name: string;
+  is_sample: boolean;
+}
+
+export interface Collection {
+  id: string;
+}
+
+export interface Dashboard {
+  id: number;
+  name: string;
+}
+
+export interface DatabaseCandidate {
+  tables: TableCandidate[];
+}
+
+export interface TableCandidate {
+  title: string;
+  url: string;
+}
```

Comment:
WDYT about creating a top-level module like `metabase/types` to contain common types like User, Card, Database, Collection, Dashboard, etc?

Reply:
I keep this and mind and will migrate this file when we have a decision.

Reply:
Yeah, totally agree here 👍